controlnext/templates/controlnext/selectbasin.html

```diff
@@ -0,0 +1,36 @@
+{% extends 'controlnext/basin_detail.html' %}
+{% load i18n %}
+
+{% block content %}
+<div class="container" style="height:150px">
+</div>
+<div class="container">
+    <div class="row">
+      <div class="col-md-4">
+      </div>
+      <div class="col-md-4">
+        <div class="panel panel-default">
+
+          <div class="panel-heading">Meerdere bassins beschikbaar. Kies het
+              gewenste Bassin.</div>
+
+          <div class="btn-group-vertical panel-body" role="group"
+               style="width:100%;">
+            {% for random_url_slug, basin_name in view.buttons %}
+              <a class="btn btn-default"
+                 href="{% url controlnext-basin random_url_slug %}">{{ basin_name }}</a>
+            {% endfor %}
+          </div>
+        </div>
+        <a class="btn btn-info" role="button" style="width:100%;"
+           href="/logout/?next=/">logout</a>
+      </div>
+    </div>
+</div>
+{% endblock content %}
+
+{% block constants %}
```

Comment:
Mee eens, het beste zou zijn een base template te maken en daar met beide templates van over te erven. Dat was wat intensief en past niet binnen de scope van deze klus. Nu laat ik het even voor wat het is.